src/scripts/docker_buildx.sh

```diff
@@ -66,6 +66,9 @@ if [ "${ORB_BOOL_SKIP_WHEN_TAGS_EXIST}" -eq "0" ] || [[ "${ORB_BOOL_SKIP_WHEN_TA
       docker --context builder buildx create --name DLC_builder --use
     fi
     context_args="--context builder"
+  # if no builder instance is currently used, create one
+  elif ! docker buildx inspect | grep "default * docker"; then 
```

Comment:
It's a good practice to suppress grep's output to stdout when using it in conditional checks.

```suggestion
  elif ! docker buildx inspect | grep -q "default * docker"; then 
```